src/components/leftDrawer/leftDrawer.scss

```diff
@@ -3,7 +3,7 @@
 .Flier {
     width: 100%;
     height: 280px;
-    background-color: #3d90b6;
+    background-color: $light-gold;
```

Comment:
Doesn't the old hex color code translate to light blue? Are we switching up the left drawer color this year?

Reply:
The new logo has a transparent background, so the color here will show as the fill behind.
I originally switched it to light gold, but then switched it back to light blue. Ill ask Holly what she thinks tomorrow...

<img width="648" alt="Screen Shot 2019-07-10 at 9 29 04 PM" src="https://user-images.githubusercontent.com/1947857/61022248-c9871200-a359-11e9-847a-af9c33cc8722.png">